companies/ellevest.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Ellevest
+tags: ["finance"]
+image: ellevest.png
+blm_statements:
+  - url: https://twitter.com/Ellevest/status/1266435834716225536
+    date_posted: 2020-05-29
+    type: Tweet
+    raw_text: >
+        Silence is a privilege. And yet, we have the power to make sure the world knows Ahmaud Arbery, Breonna Taylor, George Floyd, and the unthinkable number of Black lives lost mattered. We all have power to do something.
```

Comment:
```suggestion
      Silence is a privilege. And yet, we have the power to make sure the world knows Ahmaud Arbery, Breonna Taylor, George Floyd, and the unthinkable number of Black lives lost mattered. We all have power to do something.
```

indent 6 spaces rather than 8